tests/qgis-expected.yaml

```diff
@@ -76,31 +76,56 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/name: apache
-      app.kubernetes.io/instance: test
+      app.kubernetes.io/instance: qgis
       app.kubernetes.io/component: main
   template:
     metadata:
       labels:
         app.kubernetes.io/name: apache
-        app.kubernetes.io/instance: test
+        app.kubernetes.io/instance: qgis
         app.kubernetes.io/component: main
     spec:
 
       serviceAccountName: qgis
       securityContext:
-        readOnlyRootFilesystem: true
         runAsNonRoot: true
         runAsUser: 33
       affinity:
         {}
       initContainers:
         - name: config
           securityContext:
-            readOnlyRootFilesystem: true
             runAsNonRoot: true
             runAsUser: 33
-          image: "camptocamp/my-config-image:latest"
+          image: "camptocamp/geomapfishdemo-config:prod-2-7"
           imagePullPolicy: IfNotPresent
+          env:
+              - name: "PGDATABASE"
+                value: "postgres"
+              - name: "PGHOST"
+                value: "test-pg-postgresql"
+              - name: "PGHOST_SLAVE"
+                value: "test-pg-postgresql"
+              - name: "PGPASSWORD"
+                value: "mySuperTestingPassword"
```

Comment:
the PG credentials for testing are hard-coded here and in c2cciutils. Not really a problem, but we must keep that in mind. But I would not create a gopass thing for that actually...